packages/lodestar-cli/src/cmds/beacon/cmds/init/index.ts

```diff
@@ -13,8 +14,15 @@ export const description = "Initialize lodestar beacon node";
 export function builder(yargs: Argv<IBeaconArgs>): Argv<IBeaconArgs> {
   yargs.options(beaconInitOptions);
   const args = parseArgs(yargs) as IBeaconInitArgs;
-  if (args.templateConfigFile) {
-    yargs.config(readBeaconConfig(args.templateConfigFile));
+
+  if (args.chain && args.chain.genesisStateFile) {
+    fs.readFileSync(args.chain.genesisStateFile, "utf8");
+  }
+
```

Comment:
This check shouldn't be done here.
Check beacon/cmds/run/run.ts. Add a try/catch around the genesis state file `fs.promises.readFile` and catch, throw a YargsError with a descriptive error.

Reply:
after this PR is done, i think i might try going after #1108 and maybe just include the fix for this in that PR instead

Reply:
so for now, just gonna use this PR for template-config-file fix